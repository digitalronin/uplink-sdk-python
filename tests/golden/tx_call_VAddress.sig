r = 115136800820456833737994126771386015026287095034625623644186278108926690779567
s = 54988379004691329229286851311814037053464253567598850606302804345331763557675
