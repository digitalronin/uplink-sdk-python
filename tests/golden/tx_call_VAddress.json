{
    "header": {
        "tag": "TxContract",
        "contents": {
            "tag": "Call",
            "contents": {
                "address": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve",
                "method": "get",
                "args": [
                    {
                        "contents": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve",
                        "tag": "VAddress"
                    }
                ]
            }
        }
    },
    "signature": "AE4xMTUxMzY4MDA4MjA0NTY4MzM3Mzc5OTQxMjY3NzEzODYwMTUwMjYyODcwOTUwMzQ2MjU2MjM2NDQxODYyNzgxMDg5MjY2OTA3Nzk1Njc6AE01NDk4ODM3OTAwNDY5MTMyOTIyOTI4Njg1MTMxMTgxNDAzNzA1MzQ2NDI1MzU2NzU5ODg1MDYwNjMwMjgwNDM0NTMzMTc2MzU1NzY3NQ==",
    "timestamp": 1231006505,
    "origin": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve",
    "to": null
}